cypress/page/index.js

```diff
@@ -1,9 +1,20 @@
+// eslint-disable-next-line import/no-unresolved
 export { MenuContentPage } from "./menu-content.page";
+// eslint-disable-next-line import/no-unresolved
 export { PLPPage } from "./plp.page";
+// eslint-disable-next-line import/no-unresolved
 export { CartPage } from "./cart.page";
+// eslint-disable-next-line import/no-unresolved
 export { SigninPage } from "./signin.page";
+// eslint-disable-next-line import/no-unresolved
 export { AddressesPage } from "./addresses.page";
+// eslint-disable-next-line import/no-unresolved
 export { ShippingPage } from "./shipping.page";
+// eslint-disable-next-line import/no-unresolved
 export { PaymentPage } from "./payment.page";
+// eslint-disable-next-line import/no-unresolved
 export { ConfirmPaymentPage } from "./confirmpayment.page";
+// eslint-disable-next-line import/no-unresolved
 export { OrderConfirmationPage } from "./orderconfirmation.page";
+// eslint-disable-next-line import/no-unresolved
+export { DressesListPage } from "./dresseslist.page";
```

Comment:
the index file should have the extension `.ts` instead of `.js`, probably that will help with the eslint warning